Directory.Packages.props

```
@@ -7,30 +7,30 @@
   <ItemGroup Label="Package Versions. AutoUpdate">
     <PackageVersion Include="MSTest.TestAdapter" Version="3.1.1" />
     <PackageVersion Include="MSTest.TestFramework" Version="3.1.1" />
-    <PackageVersion Include="Microsoft.Bcl.AsyncInterfaces" Version="7.0.0" />
+    <PackageVersion Include="Microsoft.Bcl.AsyncInterfaces" Version="8.0.0" />

```

Dual targeting Net7 & Net8 will be done over coming weeks.
```suggestion
    <PackageVersion Include="Microsoft.Bcl.AsyncInterfaces" Version="7.0.0" />

```